python-lib/hubspot/writer.py

```diff
@@ -0,0 +1,19 @@
+import json, logging
+from pandas.io.json import json_normalize
+
+logger = logging.getLogger(__name__)
+
+def write_data(list_jsons, output_dataset, format_output):  
+    if format_output == 'JSON':
+        writer = output_dataset.get_writer()
+        output_dataset.write_schema([{"name": "object","type": "string"}])
+        for list_objects in list_jsons:
+            print(list_objects)
+            writer.write_row_array([json.dumps(list_objects)])
+        logger.info("Writing to output")
+
+        writer.close()
+    elif format_output == 'Readable with columns':
+        output_result = json_normalize(list_jsons)
+        output_dataset.write_with_schema(output_result)
```

Comment:
Using `write_with_schema(json_normalize(list_jsons))` might lead to out-of-memory issues (even if we use the generator as previously mentioned). We can accept this (as we have the workaround to use the JSON output format) or we can rework this part to work with `write_row_array()`

Reply:
Done!